### PR TITLE
[new release] timere.0.2.0

### PR DESCRIPTION
**Pull request by darrenldl**
OCaml date time handling and reasoning library

Homepage: https://github.com/daypack-dev/timere

### CODE REVIEW

File: packages/timere/timere.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "diet" {with-test}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+url {
+  src:
+    "https://github.com/daypack-dev/timere/releases/download/v0.2.0/v0.2.0.tar.gz"
+  checksum:
+    "1fa196c697fb5aec94cd39dbd0fd341d5eb41aff5422ad8a06c22cf93e1782b7"
```
Comment (mseri) [inline packages/timere/timere.0.2.0/opam:51]:
```suggestion
    "sha256=1fa196c697fb5aec94cd39dbd0fd341d5eb41aff5422ad8a06c22cf93e1782b7"
```

File: packages/timere/timere.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - Evaluate (sub)expressions with a different time zone (e.g. intersection of 9am to 5pm of Sydney and 9am to 5pm of New York)
+"""
+maintainer: "Darren Ldl <darrenldldev@gmail.com>"
+authors: "Daypack developers"
+license: "MIT"
+homepage: "https://github.com/daypack-dev/timere"
+bug-reports: "https://github.com/daypack-dev/timere/issues"
+dev-repo: "git+https://github.com/daypack-dev/timere"
+depends: [
+  "ocaml" {>= "4.06.1"}
```
Comment (mseri) [inline packages/timere/timere.0.2.0/opam:27]:
```suggestion
  "ocaml" {>= "4.08"}
```
Module `Int` was introduced in 4.08

Comment (mseri) [reply packages/timere/timere.0.2.0/opam:27]: Are you ok with this, or you want to update the code to support 4.06?

Comment (darrenldl) [reply packages/timere/timere.0.2.0/opam]: I'll fix the code - forgot to test the build on 4.06.1

**Comment by darrenldl** [comment]
Closing this one, I'll resubmit another PR with the fixes, thanks @mseri @kit-ty-kate

**Pull request closed without merging**